nifi-nar-bundles/nifi-hive-bundle/nifi-hive-processors/pom.xml

```diff
@@ -121,6 +121,11 @@
             <artifactId>commons-text</artifactId>
             <version>1.8</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-dbcp2</artifactId>
+            <version>2.7.0</version>
```

Comment:
The latest version of Apache Commons DBCP2 appears to be 2.8.0, can this be updated? It appears that other components use version 2.7.0, so if this update works as expected, those other references could also be updated.

Reply:
Thanks for the comment @exceptionfactory . I'd go with 2.7.0 for this change to touch as few places as possible and create a follow-up ticket to upgrade Apache Commons DBCP2 NiFi-wide. What do you think?

Reply:
Thanks for following up @adenes, it might be easier to make the changes for these particular components in this pull request, but updating all references in a separate PR also works.